README.md

```diff
@@ -110,7 +110,7 @@ used to solve the given problem.
 
 ### Detailed design discussion
 
-For the details of the API, see [wasi-nn.wit.md](wasi-nn.wit.md).
+For the details of the API, see [wasi-nn.wit.md](legacy_wit/wasi-nn.wit.md).
```

Comment:
unclear to me why this file was renamed and why the pointer is to it instead of to wit/wasi-nn.wit

Reply:
I will just delete it and point it at the new file. If someone needs access they can pull from git history.